<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addWorkloadWindow</class>
 <widget class="QDialog" name="addWorkloadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>841</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="constantRdn">
      <property name="text">
       <string>Constant</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="uniformRdn">
      <property name="text">
       <string>Uniform</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Two Stage Unfirom</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>791</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Minimum Value</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Maximum value</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Computing offload</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>791</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Processing size (MFLOPS)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="minProcSizeSpin">
      <property name="maximum">
       <double>99999999999999991611392.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="maxProcSizeSpin">
      <property name="maximum">
       <double>100000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="computingOffloadSpin">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WORKLOAD CONFIGURATION</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>791</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Communication size (MBITS)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="minCommSizeSpin">
      <property name="maximum">
       <double>10000000000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="maxCommSizeSpin">
      <property name="maximum">
       <double>999999999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>193</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>801</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Number of tasks:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="numberTasksSPin">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>791</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Interarrival distribution:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="poisonRdn">
      <property name="text">
       <string>Poisson</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="weibullRdn">
      <property name="text">
       <string>Weibull</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="exponentialRdn">
      <property name="text">
       <string>Exponential</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="fixedRdn">
      <property name="text">
       <string>Fixed</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>205</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lambda_label">
      <property name="text">
       <string>Lambda</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="shape_label">
      <property name="text">
       <string>Shape:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="firstValueSpinBox">
      <property name="maximum">
       <double>1000000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="secondValueSpinBox">
      <property name="maximum">
       <double>99999999999999991611392.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>570</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
